templates/common/_base/files/kubelet-auto-sizing.yaml

```diff
@@ -65,7 +66,7 @@ contents:
             recommended_systemreserved_cpu=$(echo $recommended_systemreserved_cpu 0.01 | awk '{print $1 + $2}')
             total_cpu=$((total_cpu-2))
         fi
-        if (($total_cpu >= 4)); then # 0.25% of any cores above 4 cores
+        if (($total_cpu >= 0)); then # 0.25% of any cores above 4 cores
```

Comment:
This was a bug too. Above 4 cores, without this fix we would start to slightly deviate away from the desired output.

Reply:
```
Core       Reserved Value
1              0.06
2              0.07
3              0.075
4              0.08
5              0.0825

after 5, keep adding 0.0025 for per cpu core
```